docs/index.yaml

```diff
@@ -149,9 +149,20 @@ entries:
     urls:
     - https://cyberark.github.io/helm-charts/conjur-config-namespace-prep-0.1.0.tgz
     version: 0.1.0
+  conjur-k8s-csi-provider:
+  - apiVersion: v1
+    created: "2023-12-29T13:37:23.980304-07:00"
+    description: A Helm chart for deploying CyberArk Conjur's CSI Driver Provider
+    digest: 49d2841770a068cdeeb0925a53fac8381e880695e515b6d12beee18600f6c0d4
+    home: https://github.com/cyberark/conjur-k8s-csi-provider
```

Comment:
Is it okay that this doesn't exist yet?

Reply:
Yep it's an optional attribute according to helm
`home: The URL of this projects home page (optional)`